desktop-widgets/downloadfromdivecomputer.cpp

```diff
@@ -1,5 +1,8 @@
 // SPDX-License-Identifier: GPL-2.0
 #include "desktop-widgets/downloadfromdivecomputer.h"
+#include "core/qthelper.h"
+#include "desktop-widgets/mainwindow.h"
+#include "desktop-widgets/command.h"
```

Comment:
given the one-liner change, it's surprising that you'd need all these new include files...